app/services/doc_auth/socure/responses/docv_result_response.rb

```diff
@@ -89,16 +85,34 @@ def extra_attributes
           }
         end
 
+        def successful_result?
+          socure_succeeded? && pii_valid?
+        end
+
+        alias_method :doc_auth_success?, :successful_result?
```

Comment:
```suggestion
        alias_method :doc_auth_success?, :socure_succeeded
```

Reply:
Nope. That's not the correct logic. Look at where `doc_auth_success?` is used; it really does mean "are we accepting this ID", not "did <vendor> say it was good."

The name in `extra_attributes` is an unfortunate collision; _it_ means "did the vendor say OK".

And the above change doesn't work anyway, because the method needs to be named `socure_succeeded`, not `socure_succeeded?`, but lint won't let me have a predicate method that isn't named `*?`.

Been down that road.